### DISCUSSION TITLE
Is it possible to configure position detection only on opening?

### DISCUSSION THREAD

**Discussion by cairon666** [floating-ui/floating-ui#2807] [Q&A] [answered]
Hey everyone

I attempted to create a Select with the search, but encountered an issue where the dropdown jumps when it is positioned on top and filtering is initiated. If you use availableHeight the jump will only occur at one time. Is it possible to configure position detection only on opening?

Example on video and  [CodeSandbox](https://codesandbox.io/p/sandbox/nice-cherry-qxsjn3?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clt5mftjm00073b6kfcrqynh7%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clt5mftjl00033b6kzfachd1y%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clt5mftjl00043b6kojw935gj%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clt5mftjm00063b6kfvuu4ixu%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clt5mftjl00033b6kzfachd1y%2522%253A%257B%2522id%2522%253A%2522clt5mftjl00033b6kzfachd1y%2522%252C%2522tabs%2522%253A%255B%255D%257D%252C%2522clt5mftjm00063b6kfvuu4ixu%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clt5mftjm00053b6k2gw34dvl%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522clt5mftjm00063b6kfvuu4ixu%2522%252C%2522activeTabId%2522%253A%2522clt5mftjm00053b6k2gw34dvl%2522%257D%252C%2522clt5mftjl00043b6kojw935gj%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clt5mftjl00043b6kojw935gj%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D):

https://github.com/floating-ui/floating-ui/assets/59088961/69fac93a-2a3b-4685-a1ff-41e5ee876e93

When using availableHeight:

https://github.com/floating-ui/floating-ui/assets/59088961/ece34a97-b12e-402b-98f1-8fe887ed3e88

**Comment by atomiks** [comment, marked as answer]
Yes: https://codesandbox.io/p/sandbox/floating-ui-flip-only-on-open-q86cny?file=%2Fsrc%2FApp.tsx%3A40%2C25

Previous discussion: https://github.com/floating-ui/floating-ui/issues/2222

**Reply by cairon666** [reply]
Thanks!
